guides/md/keras_cv/retina_net_overview.md

```diff
@@ -301,7 +334,7 @@ model.save_weights(CHECKPOINT_PATH)
 
 <div class="k-default-codeblock">
 ```
-313/313 [==============================] - 145s 424ms/step - loss: 17.2746 - classification_loss: 10.0500 - box_loss: 6.8747 - val_Mean Average Precision: 0.0105 - val_Recall: 0.0744 - val_loss: 11.2469 - val_classification_loss: 5.2460 - val_box_loss: 5.6515 - val_regularization_loss: 0.3493 - lr: 0.0100
+313/313 [==============================] - 162s 481ms/step - loss: 12.5135 - classification_loss: 6.3680 - regularization_loss: 0.0000e+00 - box_loss: 6.1454 - val_Mean Average Precision: 0.0151 - val_Recall: 0.0432 - val_loss: 10.3468 - val_classification_loss: 5.1704 - val_box_loss: 5.1764 - val_regularization_loss: 0.0000e+00 - lr: 0.0100
```

Comment:
Why is there an all-zero regularization loss?

Reply:
Hmm, I manually separated this out in the RetinaNet to ensure that it didn't leak into classification_loss or box_loss in the metrics.  Is this a bad UX?

Reply:
But why is it 0? If there is no regularization loss, we should not display it

Reply:
Yeah, I'll fix this.  https://github.com/keras-team/keras-cv/issues/821